### ISSUE TITLE
Question regarding the GeoClustering and GeoCurriculum

### ISSUE THREAD

**Issue by yl-wang996** [PKU-EPIC/UniDexGrasp2#13]
I haven't found which part of the code related to using k-means for GeoClustering, and further curriculum learning.

**Comment by yiqiaoqingyuyu** [comment]
I haven't found it either. Can the author provide any help? @wkwan7 @geng-haoran

**Comment by hshhahn** [comment]
Hi, did you figure this out?
I concluded that the code only contains the PPO baseline (which performs way better than the authors report) and no terms regarding GeoClustering.

**Comment by sanqianfengliu** [comment]
yes,i also can't find  GeoClustering and GeoCurriculum. so can we train a Generalist 𝑺𝑮0 by GeoCurriculum

**Comment by huamo555** [comment]
> Hi, did you figure this out? I concluded that the code only contains the PPO baseline (which performs way better than the authors report) and no terms regarding GeoClustering.

How do I test it? I can't find the entry for inputting model weights. How can I add the model weights given by the author?

**Comment by huamo555** [comment]
How do I test it? I can't find the entry for inputting model weights. How can I add the model weights given by the author?
